Projects/Embark/Sources/Viewables/Actions/AddressAutocompleteAction/AddressRow.swift

```diff
@@ -0,0 +1,60 @@
+import Flow
+import Form
+import Foundation
+import UIKit
+import hCore
+import hCoreUI
+
+struct AddressRow: Hashable {
+	let id = UUID()
+
+	static func == (lhs: AddressRow, rhs: AddressRow) -> Bool {
+		return lhs.id == rhs.id
+	}
+
+	func hash(into hasher: inout Hasher) { hasher.combine(id) }
+
+	let suggestion: AddressSuggestion
+	let addressLine: String
+	let postalLine: String?
+	var cellHeight: CGFloat { postalLine != nil ? 66 : 54 }
```

Comment:
you can use table views automatic cell height for this. Whenever the cell has a detail label it will adjust. I guess this useful if you want them to all be the same?